src/components/Delegates/DelegateCardList/DelegateContent.tsx

```diff
@@ -0,0 +1,42 @@
+"use client";
+
+import { useLayout } from "@/contexts/LayoutContext";
+import DelegateCardList from "./DelegateCardList";
+import DelegateTable from "./DelegateTable";
+import { PaginatedResult, PaginationParams } from "@/app/lib/pagination";
+import { DelegateChunk } from "@/app/api/common/delegates/delegate";
+import { Delegation } from "@/app/api/common/delegations/delegation";
+
+interface Props {
+  isDelegatesCitizensFetching: boolean;
+  initialDelegates: PaginatedResult<DelegateChunk[]>;
+  fetchDelegates: (
+    pagination: PaginationParams,
+    seed?: number
+  ) => Promise<PaginatedResult<DelegateChunk[]>>;
+  fetchDelegators: (addressOrENSName: string) => Promise<Delegation[] | null>;
+}
+
+export default function DelegateContent({
+  initialDelegates,
+  fetchDelegates,
+  isDelegatesCitizensFetching,
+  fetchDelegators,
+}: Props) {
+  const { layout } = useLayout();
+
+  return layout === "grid" ? (
+    <DelegateCardList
```

Comment:
https://github.com/user-attachments/assets/239354db-c183-42a5-83e2-f4100d8dab04

seeing an issue -- maybe in this component? 
I get the app to crash if I start on list view -> go to citizens tab -> back to list view

Reply:
For some reason the video is no longer loading. Tried to repro this but I can't find the "Citizens" tab on ENS. Which chain is it available on?